Ontodia/ckanext-discourse
# [PERF-71] Disable urllib3 warnings

## [Ticket](https://opengovinc.atlassian.net/browse/PERF-71)

## Description

This PR updates the discourse extension to disable warnings from urllib3 that appear in the CKAN logs.
Currently the following message appears in the log frequently:
```
InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised.
```

## Testing
- Install the ckanext-discourse extension and checkout the 'discourse-api' branch.
```
git clone https://github.com/OpenGov-OpenData/ckanext-discourse.git
cd ckanext-discourse
git checkout discourse-api
python setup.py develop
```
- Add discourse to the plugin list and add extension config to the CKAN ini config file
```
ckan.plugins = ... contact discourse pages ...

discourse.url = https://dev.discourse.org/
discourse.username = data-bot
discourse.api_key = abcd1234
discourse.ckan_category = c/open-data
discourse.category_id = 47
discourse_count_cache_age = 60
discourse.debug = false
discourse.metadata_fields = "temporal_coverage geo_coverage geographic_granularity"
```
- Start CKAN and navigate to the dataset page, click on a dataset, and navigate back to the dataset page
```
2020-02-25 15:24:13,599 INFO  [ckan.lib.base]  /dataset/tax-expenditures-by-category render time 0.546 seconds
2020-02-25 15:24:13,767 INFO  [ckan.lib.base]  /api/i18n/en render time 0.003 seconds
/usr/lib/ckan/default/local/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:838: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/security.html
  InsecureRequestWarning)
/usr/lib/ckan/default/local/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:838: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/security.html
  InsecureRequestWarning)
2020-02-25 15:24:16,861 INFO  [ckan.lib.base]  /dataset render time 1.115 seconds
```
- Checkout this PR, restart CKAN, and repeat the last step. The warning message should disappear.
```
2020-02-25 15:25:38,990 INFO  [ckan.lib.base]  /dataset/c-6a-and-c-6b-apportionment-of-state-net-income-for-c-and-s-corporations render time 0.464 seconds
2020-02-25 15:25:39,178 INFO  [ckan.lib.base]  /api/i18n/en render time 0.003 seconds
2020-02-25 15:25:41,024 INFO  [ckan.lib.base]  /dataset render time 1.121 seconds
```
